### ISSUE TITLE
Package currently depends on ansible-runner src and not the PyPI ansible-runner

### ISSUE THREAD

**Issue by anshulbehl** [ansible/ansible-navigator#249]
On line https://github.com/ansible/ansible-navigator/blob/main/setup.cfg#L36
I am getting an error when I try to just pip install ansible-navigator

```
ERROR: Packages installed from PyPI cannot depend on packages which are not also hosted on PyPI.
ansible-navigator depends on ansible-runner@ git+https://github.com/ansible/ansible-runner.git@devel#egg=ansible-runner-devel 
```

**Comment by matburt** [comment]
It will be this way until Runner 2.0 releases and is available on pypi

**Comment by cidrblock** [comment]
Fixed with version 0.6.2
